src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -0,0 +1,223 @@
+[YouTube link](https://www.youtube.com/watch?v=dQw4w9WgXcQ) on a website like the one you read this
+very article on right now on an iOS device that has the
+[YouTube iOS app](https://apps.apple.com/us/app/youtube-watch-listen-stream/id544007664) installed
+results in the link to be opened in the app, not the browser. The same is true for Android and the
+[YouTube Android app](https://play.google.com/store/apps/details?id=com.google.android.youtube&hl=en&gl=US).
+But when you install the [YouTube PWA](https://www.youtube.com/) on a macOS computer and click a
+link, it opens in… 🥁 a browser tab.
+
+But it gets more complex. What if the link appears not in a website, but in a chat message that you
+receive in one of Google's manifold chat apps? On desktop operating systems that have the notion of
+separate app windows, if the app is open already, should a new window or tab be created for each
+link click that is captured? When you think about it, there are many ways links and navigations can
+be captured, including, but not limited to, the following examples:
+
+1. Clicked links from other web pages.
+1. URL launches from a platform-specific app in the operating system.
+1. Navigations originating from the [App Shortcuts API](/app-shortcuts/)
+1. Links that go through (URL protocol handlers)[/url-protocol-handler/]
+1. Navigations caused by [file handlers](/file-handling/)
+1. Navigations caused by the [Share Target API](/web-share-target/)
+1. …and others
+
+Declarative Link Capturing is a proposal for a Web Application Manifest property called
+`"capture_links"` that lets developers determine declaratively what should happen when the browser
+is asked to navigate to a URL that is within the application's navigation scope, from a context
+outside of the navigation scope. This proposal does not apply if the user is already within the
+navigation scope (for instance, if the user has a browser tab open that is within scope, and clicks
+an internal link).
+
+{% Aside 'key-term' %} The [navigation scope](https://web.dev/add-manifest/#scope) of a Web
+Application Manifest is the `"scope"` item of a processed manifest. The navigation scope restricts
+the set of URLs to which an application context can be navigated while the manifest is applied. If
+the `"scope"` member is not present in the manifest, it defaults to the parent path of the
+`"start_url"` member. {% endAside %}
+
+Some special conditions like middle-clicking a link (or right-clicking and then "open in new tab")
+would typically not trigger the link capturing behavior. The fact whether a link is `target=_self`
+or `target=_blank` does not matter, so that links clicked in a browser window (or window of a
+different PWA) would be opened in the PWA even if they would normally cause a navigation within the
+same tab.
+
+## Suggested use cases
+
+Examples of sites that may use this API include:
+
+- Single-window PWAs where the developer prefers to only have a single instance of the app open at
```

Comment:
I'd like to add another use case, _above_ single-window. Simply PWAs that want to open a window, rather than a browser tab, when the user clicks on a link to them.

Note: Even though this is literally the description of the API, I think it's important to call it out separately and _before_ single-window (otherwise, this implies that it's only useful for single-window mode).

I actually have a big concern here that we're going to give sites the option to go single-window mode, and EVERYONE will ask for that, whether it makes sense or not, because that's how people have become accustomed to using apps on mobile. I really really want developers to think about whether they actually want to restrict usage to a single open context at a time. In a desktop environment, it makes total sense to have multiple copies of most applications open at a time. I want to discourage developers from using `existing-client-*` unless they have a good reason.

Reply:
Added, and called out that multi-window apps are very much a valid option.